feature/mtu/largeippacket/otg_tests/large_ip_packet_transmission/README.md

```diff
@@ -7,31 +7,107 @@ IPv4 and IPv6 packet sizes are sent over them.
+    successful transit.
+
+* MTU-1.3.2: Test w/ tunnel interfaces and forwarding plane via the physical and bundled interfaces
+  * Configure DUT with a GRE tunnel interface.
+    * Tunnel interface uses /32 tunnel destination and loopback interface as the source of the tunnel.
+    * Ensure MTU configured on this tunnel interface is in context to the MTU on the egress physical/bundle interface. It is acceptable if the implementation doesn't support explicit MTU config on tunnel interfaces. Idea is for the tunnel interface to respect the egress physical/bundle interface MTU config.
+  * Configure a static route for the tunnel end point destination pointing at an exit interface.
+  * Pick 2 different /24 IPv4 and /64 IPv6 subnets each to emulate destination of payload traffic to be tunneled. Note: This is for the destination prefixes of the flows generated by ATE:Port1.
+    * Ensure you have static routes in place for the inner header destination to point at the exit interface (Physical/Bundle)
+  * Run traffic flows of the following size over IPv4 and IPv6. For each scenario, the MTU configured on the physical/bundle interfaces plus the tunnel interface must be the same.
+    * 1500 Bytes
+    * 2000 Bytes
+    * 4000 Bytes
+    * 9202 Bytes
+    * 9202 Bytes
+  * Assert ATE reports packets sent and received count are the same, indicating no fragmentation, and 
+    successful transit.
+  * Run the above for GUE tunnel interface
 
 ## OpenConfig Path and RPC Coverage
 
 The below yaml defines the OC paths intended to be covered by this test.  OC paths used for test setup are not listed here.
 
 ```yaml
 paths:
-  ## Config Paths ##
-  /interfaces/interface/config/mtu:
-  /interfaces/interface/subinterfaces/subinterface/ipv4/config/mtu:
-  /interfaces/interface/subinterfaces/subinterface/ipv6/config/mtu:
-
-  ## State Paths ##
-  # No coverage, validates success by checking flow statistics between ATE ports.
+  # Config Paths
+   openconfig-interfaces:
+    - interface:
+      name: 'tunnel1_if_name'
+      config:
+        name: 'tunnel1_if_name'
+        tunnel:     # configures the tunnel parameters 
+          config:
+           src: 'tunnel1_outer_ip_src'
+           dst: 'tunnel1_outer_ip_dst'
+           ttl: 'tunnel1_outer_ttl'
+           gre-key: 'tunnel1_outer_gre_key'
+          ipv4:
+            config:
+              mtu: 'tunnel1_inner_mtu'
+            addresses:
+              - address:
+                config:
+                  ip: 'tunnel1_interface_ipv4'     # For tunnel decap destination and/or route next-hop
+                  prefix-length: 'tunnel1_interface_ipv4_prefixlen'
+          ipv6:
+            config:
+              mtu: 'tunnel1_inner_mtu'
+            addresses:
+              - address:
+                  config:
+                    ip: 'tunnel1_interface_ipv6'     # For tunnel decap destination and/or route next-hop
+                    prefix-length: 'tunnel1_interface_ipv6_prefixlen'
```

Comment:
This portion of paths needs to be moved out of the heading `## OpenConfig Path and RPC Coverage
`.  THis is because our CI requires only a flat list of OC paths to be validating.

I suggest moving this canonical config to the test setup section.  (The test procedure can reference "push the config defined in the setup to the DUT" for example).

Reply:
Moving this will fix the CI check error the PR currently has

Reply:
Done!